### ISSUE TITLE
[FEAT] Add positive and negative function

### ISSUE THREAD

**Issue by AbbeyIT** [AbbeyIT/calculator#1]
The positive and negative function is the only button that has no function. Every time the user click it, the number should be able to change into positive or negative number.

**Comment by MiliMade** [comment]
Forked! I'd like to give this go as my first issue!

**Comment by AbbeyIT** [comment]
Awesome! Thank you @MiliMade for your interest. Issue is assigned to you now.

**Comment by AbbeyIT** [comment]
@MiliMade I updated the repo update yours to avoid confusion.

**Comment by AbbeyIT** [comment]
@MiliMade Are you still working on this issue?

**Comment by MiliMade** [comment]
Yes I am! I'm sorry I'm taking so long. Do you  need it done by a certain date?

**Comment by AbbeyIT** [comment]
@MiliMade No worries. I'm just checking if you're still working on it.

**Comment by MiliMade** [comment]
Hey @AbbeyIT ,  I think I got it to work and created a pull request.
